### ISSUE TITLE
[Feature Request] Reports Module

### ISSUE THREAD

**Issue by adamsewell** [freescout-helpdesk/freescout#1096]
Would it be possible to have average response and resolution times added to the reports module? I'd love to see this broken down by different techs and clients. Thanks and keep up the good work!

**Comment by vikasprogrammer** [comment]
What do you mean by different tech? Do you use a  custom label for that?

**Comment by adamsewell** [comment]
@vikasprogrammer meaning, different users. So each tech in my company has their own account. The goal, would be to have stats per account so see how long it takes for that account to respond to tickets and resolve tickets. If that makes sense.

**Comment by vikasprogrammer** [comment]
@adamsewell Yeah, makes sense now. We are continuously developing modules for freescout internally as we use it heavily, I will put as the next item in our list. Will update here once we are done.

**Comment by fulldecent** [comment]
Do you want average or median?

Average is easy to skew with just one old ticket.

**Comment by reeyon** [comment]
Hi, know that we are able to filter the report output by tag, it will show report for tickets that assigned with tags, but are we able to filter the output by Customer Company?

**Comment by vikasprogrammer** [comment]
We did this work for a client few days back, let me know if you are interested. "

We actually create a "Company" menu to enter companies and then showed company dropdown in the Customers screen. We can also do this implemention without creating a company entity and just allow you to free style company name in customers edit screen. 

We implemented PDF export option in the reporting screen too.

Let me know if you want this and will create a custom module and host it on scoutdevs.com

![company module](https://user-images.githubusercontent.com/1262327/125394128-d31f3680-e3c6-11eb-9443-3d1de9966500.png)
